5 lesson redux/src/app/reducers/index.js

```diff
@@ -0,0 +1,20 @@
+
+import { combineReducers } from "../../../../../../Library/Caches/typescript/3.6/node_modules/redux";
```

Comment:
```suggestion
import { combineReducers } from "redux";
```
поиск пакета ("redux") и так будет осуществляться в папке `node_modules`, если мы не прописываем относительный путь